openedx/tests/settings.py

```diff
@@ -18,16 +49,39 @@
     }
 }
 
+FEATURES = {}
+
 INSTALLED_APPS = (
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
+    'djcelery',
     'openedx.core.djangoapps.video_config',
+    'openedx.core.djangoapps.video_pipeline',
     'edxval',
+    'courseware',
+    'student',
+    'certificates.apps.CertificatesConfig',
+    'course_modes.apps.CourseModesConfig',
+    'lms.djangoapps.verify_student.apps.VerifyStudentConfig',
+    'openedx.core.djangoapps.dark_lang',
+    'openedx.core.djangoapps.content.course_overviews.apps.CourseOverviewsConfig',
+    'openedx.core.djangoapps.content.block_structure.apps.BlockStructureConfig',
+    'openedx.core.djangoapps.catalog',
+    'openedx.core.djangoapps.self_paced',
+    'milestones',
+    'celery_utils',
 )
 
+LMS_ROOT_URL = 'http://localhost:8000'
+
 MEDIA_ROOT = tempfile.mkdtemp()
 
+MICROSITE_BACKEND = 'microsite_configuration.backends.filebased.FilebasedMicrositeBackend'
+MICROSITE_TEMPLATE_BACKEND = 'microsite_configuration.backends.filebased.FilebasedMicrositeTemplateBackend'
+
 SECRET_KEY = 'insecure-secret-key'
 
+TRACK_MAX_EVENT = 50000
+
```

Comment:
That's a lot of missing settings!

Reply:
Yeah, what we had before was only working because some XBlocks were failing to load but only reporting it as warnings instead of errors.  It didn't seem like a great idea to leave it like that, in case there were compatibility problems hiding in those XBlocks.  Once I actually fixed those to be able to load, it started pulling in a fair bit more initialization code.